app/src/main/java/org/eyeseetea/malariacare/fragments/PlannedFragment.java

```diff
@@ -92,8 +102,8 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
     }
 
-    private void prepareUI() {
-        this.adapter = new PlannedAdapter(plannedItems,getActivity());
+    private void prepareUI(List<PlannedItem> plannedItems) {
+        this.adapter = new PlannedAdapter(this.plannedItems,getActivity());
```

Comment:
@idelcano this is a little bit confused. Is plannedItems a parameter or variable of the class?